docs/pages/access-controls/guides/headless-login.mdx

```diff
@@ -0,0 +1,237 @@
+- Performing `tsh scp` from one Teleport Node to another with Per-session MFA
+- Logging into Teleport on a machine without a WebAuthn compatible browser
+
+<Admonition type="note" title="Headless Login Support">
+  Headless login only supports following `tsh` commands:
+  
+  - `tsh ls`
+  - `tsh ssh`
+  - `tsh scp`
+
+  In the future, headless login will be extended to other `tsh` commands.
+</Admonition>
+
+## Prerequisites
+
+- A v12.2+ Teleport cluster with WebAuthn configured.
+  See the [Second Factor: WebAuthn](./webauthn.mdx) guide.
+- WebAuthn hardware device, such as YubiKey.
+- A Web browser with [WebAuthn support](
+  https://developers.yubico.com/WebAuthn/WebAuthn_Browser_Support/).
+
+A v12.2+ Teleport cluster capable of WebAuthn is automatically capable of Headless login.
+
+## Step 1/2. Initiate headless login
+
+Run a headless tsh command with the `--headless` flag. This will initiate headless login, printing a URL and `tsh` command.
+
+```code
+$ tsh ls --headless --proxy=proxy.example.com --user=alice
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/86172f78-af7c-5935-a7c1-ed06b94f17dc
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=alice --proxy=proxy.example.com 86172f78-af7c-5935-a7c1-ed06b94f17dc
+```
+
+## Step 2/2. Approve headless Login
+
+To approve the headless login, click or copy+paste the URL printed by `tsh` in your local web browser. You will be prompted to approve the login with WebAuthn verification. Once approved, your initial `tsh --headless <command>` should continue as if you had logged in locally.
+
+Unlike a standard login session, headles login sessions are only available for the lifetime of a single `tsh` request. This means that for each `tsh --headless` command, you will need to go through the headless login flow:
+
+```code 
+$ tsh ls --headless --proxy=proxy.example.com --user=alice
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/86172f78-af7c-5935-a7c1-ed06b94f17dc
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=alice --proxy=proxy.example.com 86172f78-af7c-5935-a7c1-ed06b94f17dc
+# # User approves through link
+# Node Name Address        Labels                                                                             
+# --------- -------------- -----------
+# server01  127.0.0.1:3022 arch=x86_64
+#
+$ tsh ssh --headless --proxy=proxy.example.com --user=alice server01
+# Complete headless authentication in your local web browser:
+# 
+# https://proxy.example.com:3080/web/headless/864cccd9-2425-46d9-a9f2-636387e66ebf
+# 
+# or execute this command in your local terminal:
+# 
+# tsh headless approve --user=alice --proxy=proxy.example.com 864cccd9-2425-46d9-a9f2-636387e66ebf
+# # User approves through link
+```
+
+## Additional Options
+
+### Set headless login as default
```

Comment:
So far, all instructions have been specific to end-users, i.e., they use `tsh`. The "Additional options" section is where we include configuration instructions for Teleport operators.

Since this is not an end-user guide (i.e., it's not in the "Connect your Client" section), I think we should restructure it so that these configuration options are part of the main step-by-step flow.

Reply:
The structure is based off the [passwordless guide](https://goteleport.com/docs/access-controls/guides/passwordless/) since it is a similar WebAuthn-enhancing feature that does not require any additional configuration. 

In most use cases, Teleport operators would not want to set headless as the default, which is why this is under additional options rather than the standard setup steps, so I'd refrain from highlighting it at the beginning of the guide where it may be confused as essential configuration.

Reply:
That being said, I took a stab at restructuring it with the optional configuration at the top. Let me know what you think.